tests/test_box_utils.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2020 - 2021 MONAI Consortium
```

Comment:
now we use `# Copyright (c) MONAI Consortium` instead of the `2020 - 2021`

https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#preparing-pull-requests